src/Table/Table.stories.js

```diff
@@ -13,6 +13,11 @@ import {
   WindowContent
 } from 'react95';
 
+const Wrapper = styled.div`
```

Comment:
Please remove these type of changes from this PR (moving Wrapper in stories to the top etc)

Reply:
Some components have Wrapper defined above the export already.

This was done to fix no-use-before-define errors, would you want to disable those rules?

Reply:
oh ok, I didn't notice we enforced this rule. If that's the case let's keep the changes!